### ISSUE TITLE
Suggestion: dimensioning helpers

### ISSUE THREAD

**Issue by GilDev** [camrein/EzGraver#32]
To help prepare an engraving, first, a ruler in mm (maybe select between mm and inches) on the image's 4 sides would be nice.
Better, automatically calculate image size in pixels and convert it into millimeters. The result could be shown under the image. Here's an example with the dimensions randomly positioned (they should be outside the image) and the rectangle that “crops” to the image:
![Dimensions](https://user-images.githubusercontent.com/3149748/30400040-f4441956-98d5-11e7-8eee-15141526018f.png)

**Comment by camrein** [comment]
This feature seems quite interesting. I have already thought about the first iteration with a simple ruler on the image borders like the original software. However, I didn't integrate it as I thought that the sizes might not match for all devices. You mentioned that, at least for now, the sizes for all NEJEs is 38x38 mm in issue #28. Additionally, I thought that besides the build-platform size, also the resolution (pixel / mm ratio) might differ. But as you suggested, these things should be configurable. Nevertheless, EzGraver currently only works with 512x512 images anyway.

Supporting the configuration of the dimensions will require that the rules are rendered dynamically. But I think, as long as EzGraver only works with machines that accept a 512x512 image, it can be assumed that the devices will have the same engraving areas.

**Comment by GilDev** [comment]
Exactly. And while rulers are nice, again, calculated dimensions would be a lot more practical to use. I don't see the point of adding a ruler if there's something like this available.
Once the image pixel size is determined, the dimensions' calculation is easy and would be easy to adapt to a variable bed size, it's just a cross-multiplication.
